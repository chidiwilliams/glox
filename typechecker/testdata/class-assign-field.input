class Class {}

Class.hello = "world";
