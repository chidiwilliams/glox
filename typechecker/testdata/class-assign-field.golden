type error on line 2: only instances have fields
